Automattic/wp-calypso
# Subscriber page: create PopoverMenu for subscribers

Closes #77729

## Proposed Changes

This adds the Popover component, to be used on the subscriber list page.

![CleanShot 2023-06-09 at 14 11 29@2x](https://github.com/Automattic/wp-calypso/assets/528287/a099df3f-b027-4530-8691-efbb1425abd6)

I used the `PopoverMenu` & `PopoverMenuItem` for this, and didn't change the styling, as to stay consistent to Calypso styling.

## Testing Instructions

1. Add this PR
2. Visit `http://calypso.localhost:3000/subscribers/yoursite`
3. Verify that design looks ok & that the buttons trigger the action ( change the function inside `client/my-sites/subscribers/components/subscriber-list/subscriber-list.tsx` - lines 42:43).

Design: h8tMpJlCqNFemEerU9owHI-fi-1979_65414

## Pre-merge Checklist
